var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -125,8 +127,10 @@ class Seacas(CMakePackage):
     )
 
     # Build options
-    variant("fortran", default=True, description="Compile with Fortran support")
-    variant("shared", default=True, description="Enables the build of shared libraries")
+    variant("fortran", default=not is_windows, description="Compile with Fortran support")
+    # Enable this on Windows at your own risk, SEACAS exports no symbols and so cannot be
+    # meaningfully linked against as a shared library
+    variant("shared", default=not is_windows, description="Enables the build of shared libraries")
```

Comment:
(minor) could say `conflicts(+shared, when=windows)` but this works for now

(I was also considering putting this inside `if sys.platform != windows`, but I'm not sure if `from_variant` would act sensibly on variants that aren't present, and the CMake option exists in Windows so the conflict seems cleaner)

likewise for x11

Reply:
I'll refactor for the `conflicts` approach.